managed/devops/bin/yb_platform_util_py2.py

```diff
@@ -0,0 +1,498 @@
+
+    :param url: url to be called.
+    :param auth_uuid: authentication token of the customer.
+    :param data: data for POST request.
+    :param is_delete: to identify the delete call.
+    :return: response of the API call.
+    """
+    if not is_delete:
+        request = urllib2.Request(url)
+    else:
+        request = urllib2.Request(url)
+        request.get_method = lambda: 'DELETE'
+
+    request.add_header('X-AUTH-YW-API-TOKEN', auth_uuid)
+    request.add_header('Content-Type', 'application/json; charset=utf-8')
+    if data:
+        response = urllib2.urlopen(request, json.dumps(data).encode('utf-8'))
+    else:
+        response = urllib2.urlopen(request)
+    return response
+
+
+def get_universe_details(base_url, customer_uuid, auth_uuid, universe_name, base_dir):
+    """
+    Get the universe details and store it in a json file after formatting the json.
+
+    :param base_url: base url of the platform back end.
+    :param customer_uuid: UUID of the customer.
+    :param auth_uuid: authentication token for the customer.
+    :param universe_name: Name of the universe to take a json data from.
+    :param base_dir: base directory in which the json file should be stored.
+    :return: None
+    """
+    universe = get_universe_by_name(base_url, customer_uuid, auth_uuid, universe_name)
+    if universe:
+        configure_json = create_universe_config(universe, universe["name"])
+        file = base_dir + '/' + universe["name"] + ".json"
+        with open(file, 'w') as file_obj:
+            json.dump(configure_json, file_obj)
+        response = {
+            "data": "Detail of universe has been saved to " + str(file),
+            "status": "success",
+            "error": ""
+        }
+        handle_response(response)
+    else:
+        response = {
+            "data": "Universe details not found",
+            "status": "failed",
+            "error": "Universe with " + universe_name + " is not found."
+        }
+        handle_response(response)
+
+
+def get_universe_details_by_uuid(base_url, customer_uuid, auth_uuid, universe_uuid, base_dir):
+    """
+    Get universe details from UUID and store it in json after formatting it.
+
+    :param base_url: base url of the platform back end.
+    :param customer_uuid: UUID of the customer.
+    :param auth_uuid: authentication token for the customer.
+    :param universe_uuid: UUID of the universe to take a json data from.
+    :param base_dir: base directory in which the json file should be stored.
+    :return: None
+    """
+    universe = get_universe_by_uuid(base_url, customer_uuid, auth_uuid, universe_uuid)
+    if universe:
+        configure_json = create_universe_config(universe, universe["name"])
+        file = base_dir + '/' + universe["name"] + ".json"
+        with open(file, 'w') as file_obj:
+            json.dump(configure_json, file_obj)
+        response = {
+            "data": "Detail of universe has been saved to " + str(file),
```

Comment:
Details ... have been saved

Reply:
Also from the function name it is not clear that it saves something to a file / returns data through a file.